content/kubeone/main/tutorials/creating-clusters/_index.en.md

```diff
@@ -861,9 +875,10 @@ automatically by KubeOne as of v1.0.4.**
 apiVersion: kubeone.k8c.io/v1beta2
 kind: KubeOneCluster
 versions:
-  kubernetes: '1.22.5'
+  kubernetes: '1.25.6'
 cloudProvider:
   vsphere: {}
+  external: true
```

Comment:
For vSphere we also need `csiConfig` or otherwise validation is going to fail.

Reply:
Good catch, that's actually very important.